### PR TITLE
verbatim group added

### PR DESCRIPTION
**Pull request by anatomyOFair**

### CODE REVIEW

**Review by mbg — CHANGES_REQUESTED** [review]
Looks like there is no student with ID `2100209` on the course. Maybe a typo?

**Comment by anatomyOFair** [comment]
ah yeah sorry about that. i mistakenly put in my id from an entirely different school from the past. should be 2879471

File: config/groups.yaml [hunk 1]
```diff
@@ -565,6 +565,13 @@
     - "2428040"
     - "2475521"
   id: 190025
+- name: "Verbatim"
+  members:
+    - "2879471"
+    - "2780701"
+    - "2634491"
+    - "2575121"
+=======
```
Comment (mbg) [inline config/groups.yaml:574]: This isn't valid YAML, perhaps left over from fixing a merge conflict?

**Review by mbg — CHANGES_REQUESTED** [review]
Thanks for fixing that, but there's now a problem with a line that shouldn't be in the YAML file.

**Comment by anatomyOFair** [comment]
Sorry, I'm not entirely used to git yet. That should fix it.

**Review by mbg — APPROVED** [review]
No worries! It's all good now and I will go ahead and merge it. You should see your group on Canvas shortly.

**Pull request merged**